actions/dynamicdns

```diff
@@ -111,6 +112,21 @@ doGetOpt()
             ;;
         esac
     done
+    if [ $stdin -eq 1 ];then
+        # try to use with timeout and hidden input first
+        read -s -p "enter password and hit [ENTER]: " -t 10 password
+        RET=$?
+        # return 142 means: timeout
+        if [ -z ${password} -a ${RET} -ne 142 ];then
+            # ancient bash version does not support timeouts or hidden input
+            read -p "enter password and hit [ENTER]: " password
+        fi
+        if [ ${RET} -eq 142 ];then
+            echo "Failed to read password from stdin, try -P instead"
+            exit 1
+        fi
+        pass=${password}
+    fi
 }
```

Comment:
Similar to other action scripts, can we not just take only the password without handling new lines and timeouts?  On the command line, the password will have to be given like `$ echo -n 'password' | dynamicdns ... -p`.  After all, we don't intend the users to run the actions manually and it will be lot more simpler for us to say that the entire input is the password.  Selective handling of new lines in input can lead to unforeseen problems.  Am I missing something here?